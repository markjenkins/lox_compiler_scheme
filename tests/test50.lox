{
var a;
var b=1;
}