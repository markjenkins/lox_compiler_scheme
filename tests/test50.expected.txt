OP_NIL
OP_CONSTANT 1
OP_POP
OP_POP
OP_RETURN
